src/pages/Auth/utils.ts

```diff
@@ -51,6 +55,25 @@ export function validateEventData(data: unknown): data is QRCodeEventData {
   if (typeof clientId !== 'string' || typeof scope !== 'string') {
     return false;
   }
+  const customCodeHandlers = data?.customCodeHandlers ?? {};
+  if (customCodeHandlers && !isRecord(customCodeHandlers)) {
+    return false;
+  }
+  if (isRecord(customCodeHandlers)) {
+    for (const customCodeHandler in customCodeHandlers) {
+      if (typeof customCodeHandler !== 'string') {
+        return false;
+      }
+      if (typeof customCodeHandlers[customCodeHandler] !== 'string') {
+        return false;
+      }
+      try {
+        new RegExp(customCodeHandlers[customCodeHandler]);
+      } catch {
+        return false;
```

Comment:
Can you log these errors? I'm trying to figure out why it does not work for me, and I have no visibility whatsoever right now...